.phpcs.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
```

Comment:
See https://github.com/WebDevStudios/project-template/blob/code-support/plugins/wds-project-plugin/.phpcs.xml.dist for some extra rules to consider here.